railties/lib/rails/generators/app_base.rb

```diff
@@ -293,7 +300,7 @@ def rails_gemfile_entry
         else
           [GemfileEntry.version("rails",
                             rails_version_specifier,
-                            "Bundle edge Rails instead: gem 'rails', github: 'rails/rails'")]
+                            ["The base Rails gem, where most of the magic happens!", "https://guides.rubyonrails.org/getting_started.html"])]
```

Comment:
Just a thought: I did like how I could target rails master, even when I started using rails, this comment helped me make that leap when i was ready. I realised it was not that hard or complicated.